brouter-core/src/main/java/btools/router/WaypointMatcherImpl.java

```diff
@@ -32,6 +33,7 @@ public WaypointMatcherImpl( List<MatchedWaypoint> waypoints, double maxDistance,
     this.islandPairs = islandPairs;
     for ( MatchedWaypoint mwp : waypoints )
     {
+      // TODO[Phyks]
       mwp.radius = maxDistance * 110984.; //  6378000. / 57.3;
```

Comment:
Same question about the constant value wrt Earth radius etc.

Reply:
same answer...